src/fakeredis_cluster.erl

```diff
@@ -223,12 +221,12 @@ code_change(_OldVersion, Tab, _Extra) -> {ok, Tab}.
 %% [M1, M2, M3]
 %% [{M1, S1}, {M2, S2}]
 %% [{M1, S1, S2}, {M2, S3, S4}]
-%% -> {NodeMap, SlotsMaps}
+%% -> {Nodes, SlotsMaps}
 parse_port_args(ArgList) when is_list(ArgList) ->
     Parse = [parse_slots_maps_and_nodes(PortElem) || PortElem <- ArgList],
-    {NodeMapList, SlotsMaps} = lists:unzip(Parse),
-    NodeMap = maps:from_list(lists:flatten(NodeMapList)),
-    {NodeMap, SlotsMaps}.
+    {NodesList, SlotsMaps} = lists:unzip(Parse),
+    Nodes = maps:from_list(lists:flatten(NodesList)),
```

Comment:
Is NodesList a multi-level nested list or just one-level nested? If it's just one level of nesting, it's more clear to use lists:append/1 rather than lists:flatten/1.

Reply:
it can be a single list (of masters only), and it seems that append didnt handle that